src/systems/systemstructure.jl

```diff
@@ -140,16 +140,20 @@ get_fullvars(ts::TransformationState) = ts.fullvars
 has_equations(::TransformationState) = true
 
 Base.@kwdef mutable struct SystemStructure
-    # Maps the (index of) a variable to the (index of) the variable describing
-    # its derivative.
+    """Maps the index of variable x to the index of variable D(x)."""
     var_to_diff::DiffGraph
+    """Maps the index of an equation."""
     eq_to_diff::DiffGraph
     # Can be access as
     # `graph` to automatically look at the bipartite graph
     # or as `torn` to assert that tearing has run.
+    """Graph that connects equations to variables that appear in them."""
```

Comment:
```suggestion
    """Graph that connects variables to equations that they appear in."""
```

Sources are variables and destinations are equations iirc?

Reply:
Never mind, it's equations to variables.